action.yml

```
@@ -32,6 +32,16 @@ inputs:
       options are "latest", a version like "1.2.3", or a Git commit hash reference like "cad926a2d".
     required: false
     default: "latest"
+  target-pulumi-version:
+    description: |
+      Set the version of `pulumi/pkg` and `pulumi/sdk` to depend on for bridged providers. Currently,
+      these versions inform the linked runtime and SDK generation in all languages except Java. Valid
+      options are:
+      - "", "follow": Use the same version as pulumi-terraform-bridge
```

`"follow"` doesn't work.
```suggestion
      - "": Use the same version as pulumi-terraform-bridge
```